### DISCUSSION TITLE
AI不是科技議題 而是社會議題

### DISCUSSION THREAD

**Discussion by Jerry10413** [1101-GS4719-C/Python-Community-GS4719-C#986] [General]
最近看到的一篇文章，覺得還蠻值得分享的，我先說說我自己的想法，我在大一的時候有修過AI人工智慧與社會創新，那時就覺得比起完善AI的功能，AI會對社會造成什麼樣的影響才會是最大的議題，這就像是外星人和人類是否能夠共存的道理一樣，所以當我再看到這篇文章的時候就會非常有感觸，也分享給各位看看~~

https://buzzorange.com/techorange/2019/02/20/why-social-science-is-important-to-ai/#:~:text=AI%20%E4%B8%8D%E6%98%AF%E7%A7%91%E6%8A%80%E8%AD%B0%E9%A1%8C%EF%BC%8C%E6%98%AF%E3%80%8C%E7%A4%BE%E6%9C%83%E8%AD%B0%E9%A1%8C%E3%80%8D%20AI%20%E6%98%AF%E6%9C%80%E5%85%B7%E4%BB%A3%E8%A1%A8%E6%80%A7%E7%9A%84%E5%89%B5%E6%96%B0%E7%A7%91%E6%8A%80%E6%8A%80%E8%A1%93%EF%BC%8C%E4%BC%81%E6%A5%AD%E3%80%81%E7%94%A8%E6%88%B6%E9%83%BD%E4%BB%A5%E6%96%B0%E6%8A%80%E8%A1%93%E7%9A%84%E8%A7%92%E5%BA%A6%E4%BE%86%E7%9C%8B%E5%BE%85%20AI%20%E7%9A%84%E6%87%89%E7%94%A8%E8%88%87%E7%99%BC%E5%B1%95%EF%BC%8C%E4%BD%86%E5%85%B6%E5%AF%A6,AI%20%E6%89%80%E7%94%A2%E7%94%9F%E7%9A%84%E9%81%A0%E9%81%A0%E4%B8%8D%E5%83%85%E6%98%AF%E6%8A%80%E8%A1%93%E8%AD%B0%E9%A1%8C%E8%80%8C%E5%B7%B2%E3%80%82%20%E5%A6%82%E4%BD%95%E4%BD%BF%E7%94%A8%20AI%E3%80%81%E5%A6%82%E4%BD%95%E5%BB%BA%E7%AB%8B%20AI%20%E6%A8%A1%E5%9E%8B%E5%85%B6%E5%AF%A6%E9%83%BD%E8%88%87%E7%A4%BE%E6%9C%83%E6%97%A5%E5%B8%B8%E7%94%9F%E6%B4%BB%E6%9C%89%E9%97%9C%EF%BC%8C%E5%BF%85%E9%A0%88%E8%A6%81%E5%81%9A%E6%9B%B4%E9%80%9A%E7%9B%A4%E7%9A%84%E6%80%9D%E8%80%83%E3%80%82

**Comment by ghost** [comment]
任何事情都有一體兩面，在ai的發展上，固然有技術上如何突破的問題，但在技術突破同時，也應該思考他所帶來的弊端，畢竟使用的是人，而人在面對事物時，不僅該考慮效率，還會有心情上的影響

**Comment by manipulateed** [comment]
人文的發展確實該跟科技並肩而行，致使在快速發展的科技當中，我們不會失去人的感性與人文的精神，包含道德，文化等等背景。

**Comment by BlahFish1006** [comment]
有看過一個Netflix的影集叫《黑鏡》，有點像是用預言的手法，講述科技對人性的迫害面，蠻值得一看的！
每次看完都起雞皮疙瘩，但又十分發人深省。

**Comment by dinosaur1009** [comment]
AI的發展一直以來是值得討論的議題 如何拿捏平衡也很值得思考

**Comment by Sylvie0601** [comment]
我認為AI發展也是十分需要人文社會的導正涵在旁協助，若是只有一昧的讓機器人判定就會造成像新聞所說的案例發生，如何教導AI這種文化、道德觀還是需要十分多的人文社會長才在旁協助。

**Comment by Valentinaa0214** [comment]
科技進步確實在許多方面發揮了重要作用。人類能夠更快速地傳送信息和資訊，使生產力得到大幅提升，也為人類的生活帶來了許多方便。但是，我們也不能忽略人文發展的重要性。自身文化、價值觀、思想、感情、道德等方面的發展，是人類社會進步的基礎。如果人類只重視科技進步，而忽略了人文發展，就有可能導致社會不平衡，甚至出現矛盾、衝突。

**Comment by moonnight3503** [comment]
AI居然也有歧視的問題，是因為前的數據造成不同族群被AI認定為某方的犯罪率會增加嗎?
不過AI在社會問題上也是層出不窮，像道德問題和一些侵權、個人隱私的事件，都是在科技的進步下所衍伸出來的外部效果。

**Comment by dabbie109303501** [comment]
我覺得AI歧視的問題有一部分也反映出目前存在的社會問題
當然在文章中提到的撞死老婦的行為無法原諒 需要給AI加上生死的觀念
但在AI歧視黑人和婦女的方面 我認為AI的判斷情有可原
因為AI是從統計資料中學習的 而在目前的社會中 又有多少男女同工不同酬和種族歧視的案例?
雖然與以前相比已經少很多 但仍有此現象 尤其因中國的covid19 讓近幾年的美國除了黑人歧視之外又多了黃種人歧視
在這樣的社會風氣下 又怎麼敢希望吸收那些資料教出來的AI可以公平公正不歧視?

**Comment by Kotori0501** [comment]
AI的科技真的發展的越來越快，但同時或許會有思路相同的情況發生，因此創新的出現真的很重要，或許在日後上班時，如果是需要研發產品的話，身邊同事或許會給你靈感，所以要好好的掌握人際關係，互相交流，一定可以有屬於自己的創新作品
